dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -66,12 +67,13 @@ def make_project_dataframe(self, project_config):
             # There is currently no case that needs model years or value columns.
             ld_df = self._convert_time_dimension(ld_df, project_config)
 
-        lk_df = self._remap_dimension_columns(lk_df)
         null_lk_df = self._remap_dimension_columns(null_lk_df)
         ld_df = self._remap_dimension_columns(ld_df)
         pivoted_columns = set(ld_df.columns).intersection(
             self.get_pivoted_dimension_columns_mapped_to_project()
         )
+        if SCALING_FACTOR_COLUMN in ld_df.columns:
+            ld_df = apply_scaling_factor(ld_df, pivoted_columns)
```

Comment:
This seems like an important line to cover with a test, but codecov says it's not covered yet.

Reply:
I agree. I wrote a unit test for the scaling factor logic but not an integration test with a real dataset. Sorry, I meant to call that out. It takes some time to invent datasets and check things like this. For resstock and comstock we perform independent verification of all the logic that goes into mapping a dataset to a project. The open question is whether we defer this integration test until we have a real dataset or spend time inventing one here.